src/System.Private.CoreLib/shared/System/Type.Helpers.cs

```diff
@@ -141,15 +141,15 @@ public virtual Type[] FindInterfaces(TypeFilter filter, object filterCriteria)
             return ret;
         }
 
-        public virtual MemberInfo[] FindMembers(MemberTypes memberType, BindingFlags bindingAttr, MemberFilter filter, object filterCriteria)
+        public virtual MemberInfo?[] FindMembers(MemberTypes memberType, BindingFlags bindingAttr, MemberFilter? filter, object? filterCriteria)
         {
             // Define the work arrays
-            MethodInfo[] m = null;
-            ConstructorInfo[] c = null;
-            FieldInfo[] f = null;
-            PropertyInfo[] p = null;
-            EventInfo[] e = null;
-            Type[] t = null;
+            MethodInfo?[]? m = null;
```

Comment:
We shouldn't set this ones as nullable, just don't initialize them as null :smile:

Reply:
I tried that too but below from row 270 when checkign if these arrays n!= null compiler complains unnassigned error
`if (m != null)`

Reply:
Oh I see. Makes sense. Leave it as is then.